dev/com.ibm.websphere.appserver.features/visibility/public/jwt-1.0/com.ibm.websphere.appserver.jwt-1.0.feature

```diff
@@ -5,15 +5,17 @@ visibility=public
 IBM-ShortName: jwt-1.0
 IBM-API-Package: com.ibm.websphere.security.jwt; type="ibm-api"
 Subsystem-Name: JSON Web Token 1.0
--features=com.ibm.wsspi.appserver.webBundle-1.0, \
+-features=\
+  com.ibm.wsspi.appserver.webBundle-1.0, \
   com.ibm.wsspi.appserver.webBundleSecurity-1.0, \
-  com.ibm.websphere.appserver.servlet-3.0; ibm.tolerates:="3.1, 4.0"
--jars=com.ibm.websphere.appserver.api.jwt; location:=dev/api/ibm/
--files=dev/api/ibm/javadoc/com.ibm.websphere.appserver.api.jwt_1.1-javadoc.zip
--bundles=com.ibm.ws.security.jwt, \
+  com.ibm.websphere.appserver.servlet-3.0; ibm.tolerates:="3.1, 4.0, 5.0"
+-jars=\
+  com.ibm.websphere.appserver.api.jwt; location:=dev/api/ibm/
```

Comment:
com.ibm.websphere.appserver.api.jwt will need to move to the javaee auto feature and the com.ibm.websphere.appserver.api.jwt/bnd.bnd file will need to be updated to do jakartaeeMe and a new API bundle created and reference in the jakarta auto feature.

Reply:
Made the requested changes.